hi-ml-cpath/amlt_singularity_cpath.yml

```diff
@@ -0,0 +1,31 @@
+description: Simple PyTorch job on MNIST on Singularity
```

Comment:
This file is included to give an example of what an Amulet config file should look like. Importantly, we have the `conda_yaml_file` definition and a list of multiple `jobs`